controllers/case_controller.py

```diff
@@ -51,6 +51,11 @@ def get_b_case(collection_exercise_id, business_id):
                   if case['sampleUnitType'] == 'B')
     logger.debug('Successfully retrieved B case',
                  collection_exercise_id=collection_exercise_id, business_id=business_id)
+
+    iac = get_case_iac(b_case['id'])
```

Comment:
There is an IAC flag used `get_cases_by_casegroup` above - could that be used to get the IAC instead? I'm currently getting an IAC of `null` at the moment however when that call to the case service is made, so I might be wrong!